intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_config.h

```diff
@@ -41,6 +41,7 @@ struct gpt_params {
   int32_t n_gpu_layers = 0;  // number of layers to store in VRAM
 
   // sampling parameters
+  bool do_sample = false;
```

Comment:
default is false ?
what post process we will use as default ?

Reply:
greedy search by default